pyproject.toml

```diff
@@ -1,3 +1,51 @@
+[build-system]
+requires = ["setuptools>=61"]
+build-backend = "setuptools.build_meta"
+
+[project]
+version = "v2.1.1"
+name = "PyGithub"
+authors = [{ name = "Vincent Jacques", email = "vincent@vincent-jacques.net" }]
+description = "Use the full Github API v3"
```

Comment:
The `setup.py` had a `long_description`, which ended up on pypi as the project description of a release:

https://pypi.org/project/PyGithub/

How can we get this with `pyproject.toml`?

Reply:
there is `readme = "README.md"` to use readme.md as long description

Reply:
I think that should be even better, because we then do not duplicate the content.

Reply:
I didn't get this, where is the duplicate?

Reply:
With `setup.py`, that string used to be duplicated, one copy in `README.md`, one in `setup.py`. Moving this to `pyproject.toml` is duplication is resolved because `README.md` is reused.